uint8 isadj_frame
uint16 type
int32 lf_forward
int32 ri_forward
int32 lf_back
int32 ri_back
uint64 time_stamp
